### PR TITLE
[Feature][Jdbc] Support sink ddl for postgresql

### PR DESCRIPTION
**Pull request by hawk9821**
### Purpose of this pull request
#8113 

### How was this patch tested?
68226f9c001144715804a609b3b4922

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
cc @hailin0 and @dailai

File: seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java
Comment (Hisoka-X) [inline seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java:804]: why not move this part into `MysqlDialect`

Comment (hawk9821) [reply seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java:804]: for future support for cross-database default values for ddl statements ,  here source and sink should be many-to-many , Unified abstract DefaultValueUtils class.   The default value of the upstream transmission to sink should be determined by the upstream database to determine whether it is a special default value, rather than by `isSpecialDefaultValue`  method of Sink.

Comment (Hisoka-X) [reply seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java:804]: @hailin0

**Comment by dailai** [comment]
Please add e2e for pg-cdc to jdbc pg. I think this is probably the more common user case.

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-ddl/src/test/resources/mysqlcdc_to_postgres_with_schema_change.conf
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-ddl/src/test/resources/mysqlcdc_to_postgres_with_schema_change.conf]: rebase dev

Comment (hawk9821) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-ddl/src/test/resources/mysqlcdc_to_postgres_with_schema_change.conf]: rebased

**Comment by hawk9821** [comment]
> Please add e2e for pg-cdc to jdbc pg. I think this is probably the more common user case.

It supports sink DDL for postgresql  and provides a unified jdbc sink dll template .  postgres cdc schema evolution  requires a new pr.

**Comment by hailin0** [comment]
Please update docs

https://github.com/apache/seatunnel/pull/8250/files#diff-9585a723df49ffa4f8281f070b49bbfb235e20bb5ef6b32ac78f3890a2405cad

**Comment by hawk9821** [comment]
> Please update docs
> 
> https://github.com/apache/seatunnel/pull/8250/files#diff-9585a723df49ffa4f8281f070b49bbfb235e20bb5ef6b32ac78f3890a2405cad

get, added

**Review by hailin0 — DISMISSED** [review]
LGTM

waiting for ci passed!

**Review by dailai — DISMISSED** [review]
LGTM If ci passed

**Review by hailin0 — APPROVED** [review]
LGTM If ci passed

**Review by dailai — APPROVED** [review]

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**
